internal/agent/environment.go

```diff
@@ -33,8 +33,9 @@ func (a *Agent) CreateEnvironment(ctx context.Context, req *proto.CreatEnvReques
 	// Setting up the env config
 	var envConf env.EnvConfig
 	envConf.Tag = req.EventTag
+	envConf.Type = int(req.EnvType)
 	envConf.WorkerPool = a.workerPool
-
+	log.Debug().Int("envtype", envConf.Type).Msg("making environment with type")
```

Comment:
Maybe convert the type to a string for easier debugging

Reply:
Also in proto or just here for easier readability?